pkg/services/ngalert/models/image_test.go

```diff
@@ -20,12 +20,18 @@ func TestImage_ExtendDuration(t *testing.T) {
 }
 
 func TestImage_HasExpired(t *testing.T) {
+	oldTimeNow := timeNow
+	timeNow = newFixedTimeNow()
+	t.Cleanup(func() {
+		timeNow = oldTimeNow
+	})
+
 	var i Image
-	i.ExpiresAt = time.Now().Add(time.Minute)
+	i.ExpiresAt = timeNow().Add(time.Minute)
 	assert.False(t, i.HasExpired())
-	i.ExpiresAt = time.Now()
+	i.ExpiresAt = timeNow().Add(-time.Millisecond)
```

Comment:
This test should check whether `ExpiresAt = timeNow` is considered expired. Your change is basically the same as few lines below